CoreModules/elements/base_element.hh

```diff
@@ -77,6 +77,9 @@ struct Pot : ParamElement {
 	std::string_view units = "";
 	bool integral = false;
 	uint8_t display_precision = 0;
+	constexpr static size_t MaxPosNames = 16;
+	unsigned num_pos = 16;
```

Comment:
I think the default should be 0, to indicate continuous (which aligns with normal default pots have 0 value elements in pos_names)